tests/op/extract/split/test_markdown_header_splitter.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+from uniflow.node import Node
+from uniflow.op.extract.split.markdown_header_splitter import MarkdownHeaderSplitter
+
+
+class TestMarkdownHeaderSplitter(unittest.TestCase):
+    def setUp(self):
+        self.splitter = MarkdownHeaderSplitter("test_splitter")
+
+    def test_call(self):
+        node0 = Node(name="node1", value_dict={"text": "# Header ## Content"})
+        node1 = Node(name="node1", value_dict={"text": "# Header\n## Content"})
+
+        output_nodes = self.splitter([node0, node1])
+
+        self.assertEqual(len(output_nodes), 2)
+        self.assertEqual(output_nodes[0].value_dict["text"], ["# Header ## Content"])
+        self.assertEqual(output_nodes[1].value_dict["text"], ["# Header", "## Content"])
+
+    def test_header_splitter(self):
```

Comment:
can split into two test cases if you are intended to test single_header and multiple_header:
test_header_splitter_with_single_header
test_header_splitter_with_multiple_header

Reply:
It's testing processing `Sequence[Node]`.